File: exercicios em Python/q2.py
import os

print("\nEste programa calcula a area e o perimetro de um retangulo qualquer!")
base = float(input("\nDigite o valor da base: "))
altura = float(input("\nDigite o valor da altura: "))
area = 0
area = int(base * altura)
perimetro = int(base + base + altura + altura)
print("\nO valor da area do retangulo e de ",area,", e o valor do perimetro e de ",perimetro,".")
print("\nObrigado por utilizar nosso sistema!")
os.system ("pause")

File: exercicios em C/q2.cpp
#include <stdio.h>

int main(void){
	int base, altura, perimetro, area;
	printf("\n\nEste programa determina a area e o perimetro de um retangulo qualquer.");
	printf("\nDigite o valor desejado da base do seu retangulo: ");
	scanf("%d", &base);
	printf("\nAgora digite o valor da altura do retangulo: ");
	scanf("%d", &altura);
	area = base * altura;
	perimetro = base + base + altura + altura;
	printf("\n\n----------");
	printf("\n|        | Altura %d", altura);
	printf("\n|        |");
	printf("\n----------");
	printf("\n  base %d", base);
	printf("\n\n%d * %d = %d", base, altura, area);
	printf("\n%d + %d + %d + %d = %d", base, base, altura, altura, perimetro);
	printf("\n\nSeu retangulo com base de valor %d, e altura %d, tem a area com o valor de %d, e o perimetro de %d.", base, altura, area, perimetro);
	printf("\n\nObrigado por utilizar nosso sistema!");
	return 0;
}
